### DISCUSSION TITLE
How do I correctly drive the UH630 core display while using Nootx and deleting Whatevergreen

### DISCUSSION THREAD

**Discussion by Liyouxiu01** [ChefKissInc/NootRX#101] [Q&A] [answered]
How do I correctly drive the UH630 core display while using Nootx and deleting Whatevergreen

**Comment by VisualEhrmanntraut** [comment, marked as answer]
NootRX does not have code for Intel iGPUs.

**Reply by Liyouxiu01** [reply]
But when I add PciRoot (0x0)/Pci (0x2,0x0), I can currently pull GPU acceleration up to 1.20 GHz, but the hacktool cannot obtain buffer frame information
<img width="696" alt="截屏2024-12-06 23 36 49" src="https://github.com/user-attachments/assets/7dd02530-ba15-4236-8ddb-b5998e432aec">
Intel CoffeeLake [UHD Graphics 630]: Name: display Type: Display controller Installed Driver: Yes MSI: Yes Bus: PCI Slot: Internal@0,2,0 Supplier ID: 0x8086 Device ID: 0x3e98 Subsystem Supplier ID: 0x1849 Subsystem ID: 0x3e92 Modified Version ID: 0x0000 Link Width: x0

**Reply by Liyouxiu01** [reply]
<img width="421" alt="截屏2024-12-06 23 41 25" src="https://github.com/user-attachments/assets/33d4beae-bbee-4ed9-9ba7-9f8e7e52f23f">
